### PR TITLE
docs: improved en version of sql/dql/select_into

### PR DESCRIPTION
**Pull request by michelle-qinqin**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs

* **What is the current behavior?** (You can also link to an open issue here)
* **What is the new behavior (if this is a feature change)?**
fixed sql errors
fixed syntax errors in the table of the en version
improved the syntax format in both en and zh version

### CODE REVIEW

File: docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
 # SELECT INTO
+The `SELECT INTO OUTFILE` statement is used to export the query results into a file. 
 
+```{note}
+ The [`LOAD DATA INFILE`](../dml/LOAD_DATA_STATEMENT.md) statement is complementary to `SELECT INTO OUTFILE`, which allows the user to create a table from a specified file and load data into the table.
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md]: the user => "a user" or "users"
这里的user是泛指，没有明确的指代关系

File: docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
+| delimiter          | String  | ,               | It determines the column separator of the exported file.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
Comment (lumianph) [inline docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md]: determine seems a bit weird in this context, you may change all `determine` to `define` in this table

File: docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md (hunk 1 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md]: ok~

File: docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md (hunk 2 above)
Comment (michelle-qinqin) [reply docs/en/reference/sql/dql/SELECT_INTO_STATEMENT.md]: ok~

**Pull request merged**